src/main/resources/messages.properties

```
@@ -1132,6 +1132,13 @@ faq.questions-about-the-process.how-do-i-maintain-benefits.a-renewal-form=About
 faq.questions-about-the-process.how-long-will-it-take=After I apply, how long will it take to get child care assistance?
 faq.questions-about-the-process.how-long-will-it-take.you-should-get-an-application-response=You should get an application response within 45 calendar days. This is counted from the date your completed application was received by your CCR&R.
 
+faq.questions-about-being-a-child-care-provider.header=Questions about being a child care provider
+faq.questions-about-being-a-child-care-provider.can-I-have-other-jobs=Can I have other jobs in addition to providing child care?
+faq.questions-about-being-a-child-care-provider.can-I-have-other-jobs.answer=You can have other jobs, but the work hours can't overlap. To qualify as a CCAP child care provider, you can't work another job while you are taking care of children.
+faq.questions-about-being-a-child-care-provider.how-do-ccap-payments-work=How do CCAP payments work?
+faq.questions-about-being-a-child-care-provider.how-do-ccap-payments-work.answer-1=Child care providers can be paid by paper check, direct deposit, or debit card. To find out which payment methods you can use, call the CCAP office at (309) 205-3070 Option 2.
+faq.questions-about-being-a-child-care-provider.how-do-ccap-payments-work.answer-2=For more information about CCAP payments, visit <a href="https://www.dhs.state.il.us/page.aspx?item=31637" target="_blank">this website</a>.
```

Sorry I missed this before. To track this link in mixpanel we need an id, eg:


```suggestion
faq.questions-about-being-a-child-care-provider.how-do-ccap-payments-work.answer-2=For more information about CCAP payments, visit <a data-mixpanel='link-to-dhs' href="https://www.dhs.state.il.us/page.aspx?item=31637" target="_blank">this website</a>.
```